content/blog/2021/06/2021-06-17-student-luminy-project-security.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Four students and their master project in Jenkins security"
+tags:
+- plugins
+- security
+- jenkins
+authors: 
+- jphilip
+- kguerroudj
+- qparra
+- mheyries
+opengraph:
+  image: /images/post-images/2020/08-external-fingerprint-storage-phase-3/opengraph.png
```

Comment:
We need a different opengraph image.
```suggestion
  image: /images/post-images/2020/08-external-fingerprint-storage-phase-3/opengraph.png
```

That image describes an [external fingerprint storage](https://www.jenkins.io/images/post-images/2020/08-external-fingerprint-storage-phase-3/opengraph.png) project.

Attached is a possible idea for an opengraph image (see the [original source](https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit#slide=id.ge0af3de53e_0_0) if you'd like to modify it).

![Jenkins Social Media Covers (1)](https://user-images.githubusercontent.com/156685/122239090-e5688c00-ce7d-11eb-8722-c959ea41013b.png).